src/components/diagrams/diagram-common.ts

```diff
@@ -61,13 +61,18 @@ export const styles = {
                 stroke: theme.palette.text.primary,
             },
         '& .sld-flash, .sld-lock': {
-            stroke: 'none',
             fill: theme.palette.text.primary,
+            stroke: 'none',
         },
         '& .arrow': {
             fill: theme.palette.text.primary,
         },
     }),
+    divSingleLineDiagramDevModeDisabled: (theme: Theme) => ({
```

Comment:
I think that here the naming should be independent from the devMode and describe more specifically what it does : 
```suggestion
    divSingleLineDiagramHideLock: (theme: Theme) => ({
```